Directory.Build.props

```diff
@@ -9,11 +9,11 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <HighEntropyVA>true</HighEntropyVA>
-    <LangVersion>9.0</LangVersion>
+    <LangVersion>latest</LangVersion>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <Product>Microsoft Health</Product>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <SupportedFrameworks>netcoreapp3.1;net5.0</SupportedFrameworks>
```

Comment:
Just wondering, why not keep the variable and change to `net6.0`?

Reply:
I've actually gone back and added it back. I've also figured out how to multi-target the code-gen stuff.